scripts/psake-build.ps1

```diff
@@ -50,8 +50,8 @@ Task Clean.PublishFolder {
 Task Generate.Version {
     
     $script:versionBase = get-content $baseDir\version.txt -ErrorAction SilentlyContinue
```

Comment:
do we want `join-path` for platform-insensitivity?

Reply:
I think I'll do a separate PR for Linux compatibility